CHANGELOG.md

```diff
@@ -90,10 +90,48 @@
 
 ## v1.36.0 - TBD
 
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+**BREAKING CHANGE:** The `bigtable::InstanceAdminClient` interface has changed
+significantly. Any code that extends this class or calls its experimental public
+APIs (`reset()`, `Channel()`) will be broken. For the most part, this should
+only affect customers who mock this class in their tests. Code that calls
+`bigtable::MakeInstanceAdminClient()` or
+`bigtable::CreateDefaultInstanceAdminClient()` will continue to work as before.
+
+This change will allow us to deliver new features more quickly by reducing the
+maintenance costs of the library. It also provides a better mocking interface
+for `bigtable::InstanceAdmin`. We are sorry to those that are inconvenienced by
```

Comment:
s/We are sorry to those that are inconvenienced/We apologize if you are inconvenienced/